Worker.Extensions.Sql/src/Microsoft.Azure.Functions.Worker.Extensions.Sql.csproj

```diff
@@ -22,4 +22,10 @@
 		<None Include=".\README.md" Pack="true" PackagePath="" />
 	</ItemGroup>
 
-</Project>
+	<ItemGroup>
+		<AssemblyAttribute Include="Microsoft.Azure.Functions.Worker.Extensions.Abstractions.ExtensionInformationAttribute">
+			<_Parameter1>Microsoft.Azure.WebJobs.Extensions.Sql</_Parameter1>
+			<_Parameter2>0.*-*</_Parameter2>
```

Comment:
This doesn't resolve all the tasks listed in the issue : 

1. The published extension version is now 1.*, so this should be updated correct? (which demonstrates the usefulness of it being here in the first place...)
2. You should also be creating a pipeline var and then using that instead of hardcoding it https://github.com/Azure/azure-functions-sql-extension/blob/main/builds/azure-pipelines/build-release.yml#L13 (defaulting to 99.99.99 so it continues to work for local builds)